### DISCUSSION TITLE
Domain already link to my account, but don't see it in vercel

### DISCUSSION THREAD

**Discussion by meyer0x** [vercel/community#6278] [Account Issue] [answered]
### Question

Hello,

I already have a domain linked to my account and my production site is on it so no problem. Except that I don't see it in my domains, and when I try to add it I get the error message that it's already linked to my account. I think it's because a new "team" has been automatically created with the same name as the initial one and project at the end, and the old team hasn't been deleted.

How can I manage my domain now?

**Comment by meyer0x** [comment]
my domain @amyegan i ping you because you speak at the same problem in another discussion

**Reply by younes0** [reply, marked as answer]
@meyer0x I found my "personal" domains on https://vercel.com/account/domains and moved them to my "personal team" to get the same behavior as previously

**Reply by meyer0x** [reply]
Thank you !
